shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -12,11 +12,17 @@ \section{Protocol Parameters}
 Negative values will not be allowed in UTxO outputs or reward accounts,
 and $\Z$ is only chosen over $\N$ for its additive inverses.
 
-Two global constants are defined, $\SlotsPerEpoch$ and $\SlotsPerKESPeriod$,
-representing the number of slots in an epoch/KES period (for a brief explanation
+Six global constants are defined.
+As global constants, these values can only be changed by updating the software.
+The constants $\SlotsPerEpoch$ and $\SlotsPerKESPeriod$
+represent the number of slots in an epoch/KES period (for a brief explanation
 of a KES period, see Section \ref{sec:crypto-primitives-shelley}).
-As global constants, these values can only be
-changed by updating the software.
+The constants $\SlotsPrior$ and $\StartRewards$ concern the chain stability.
+The maximum number of time a KES key can be evolved before a pool operator
+must create a new operational certificate is given by $\MaxKESEvo$.
```

Comment:
should we somehow note that a change to `\MaxKESvo` might require a software update, too, i.e., if the number of required keys grows beyond the next power of two, the format of the KES signatures would change